app/views/layouts/_footer.html.haml

```diff
@@ -6,5 +6,13 @@
         %h5.mt-2 Career Crafter Pro
         %p.small We are currently beta testing our platform. Thank you for your patience.
         %p.small If you are interested in participating as a beta tester please reach out.
-        .span
-          = link_to 'Home', root_path, class: 'btn btn-outline-light mt-3'
+        .row.justify-content-center.g-0
+          .col
+            = link_to 'About Us', about_path, class: 'btn btn-outline-light mt-3'      
+          .col
+            = link_to 'Home', root_path, class: 'btn btn-outline-light mt-3' 
+          .col
+            .btn.btn-outline-light.mt-3
+              %a{ href: 'https://www.linkedin.com/company/career-crafter-pro/', target: '_blank', style: "color: #944848"} 
+                %i.bi.bi-linkedin
+                LinkedIN
```

Comment:
for this section instead of using `.col` wrapper over each link_to - it creates large spaces between the buttons/links try wrapping everything in a `span` and remove `.col` wrappers. This will keep all 3 links on one line but also limit their spacing between each other. 

```
        .row.justify-content-center.g-0
          .span
            = link_to 'About Us', about_path, class: 'btn btn-outline-light mt-3'

            = link_to 'Home', root_path, class: 'btn btn-outline-light mt-3'

            .btn.btn-outline-light.mt-3
              %a{ href: 'https://www.linkedin.com/company/career-crafter-pro/', target: '_blank', style: "color: #944848"}
                %i.bi.bi-linkedin
                LinkedIN
```

It looks like this: 

![Screenshot 2024-08-11 at 6 15 16 PM](https://github.com/user-attachments/assets/dcb81c45-a00b-41d1-8337-d4737a027834)